docs/misconfiguration/examples/others.md

```diff
@@ -0,0 +1,22 @@
+# Others
+
+!!! hint
+    See also [Others](../../vulnerability/examples/others.md) in Vulnerability section.
+
+## File patterns
+When a directory is given as an input, Trivy will recursively look for and test all files based on file patterns.
+The default file patterns are  [here](../policy/custom.md).
+
+In addition to the default file patterns, the `--file-patterns` option takes regexp patterns to look for your files.
+For example, it may be useful when your file name of Dockerfile doesn't match the default patterns.
+
+This can be repeated for specifying multiple file patterns.
+Allowed values are here:
```

Comment:
what do you mean by "allowed values" the entire purpose of this flag is to supply my own custom value no?

Reply:
No, it controls how Trivy looks for config files. Trivy looks for config files according to file name and extension. When a user might have Dockerfile not named "Dockerfile" and have different name conventions such as `webapp.docker` and `db.docker`, Trivy will not handle the file as Dockerfile. In that case, a user can specify `--file-patterns dockerfile:.*.docker`.